### ISSUE TITLE
error with fuel

### ISSUE THREAD

**Issue by Tkaag** [Fatmice/Uranium-Power-Cont#6]
Hey 
first of all cool mod :D
in factorio 0.12.2 i get this error when putting in fuel
![image](https://cloud.githubusercontent.com/assets/4026751/9143882/4b3c0f6a-3d49-11e5-85db-8382b0cc68ef.png)

kaag101

**Comment by Fatmice** [comment]
What sort of fuel did you put in the chest?  What version are you using?  The exact step to reproduce your error?  This sort of bug report is very hard to follow-up on.

**Comment by Tkaag** [comment]
the version is 6.2
i tried with the 1.7 U-235 and the last one you can make. 
i just put the reaktor down the chest on the side put in fuel and the error came

**Comment by Fatmice** [comment]
I can not reproduce your error.  The only way I know of causing such an error is if you put a non-fuel item into the reactor port.  Make sure that fuel assemblies are from Uranium Power and nothing else.

**Comment by Tkaag** [comment]
i found my error... i forgot to turn it in to fuel..

2015-08-08 9:50 GMT+02:00 Fatmice notifications@github.com:

**Comment by Fatmice** [comment]
I will add new code to catch this user behavior. -.-'

**Comment by Fatmice** [comment]
Fixed in commit 65070de82a35db502661c9024562c8e50c69af74
